docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+:kafkacat-url-kafka: kafka/kcat-kafka/README.adoc
+:quarkus-url-kafka: kafka/quarkus-kafka/README.adoc
+:nodejs-url-kafka: kafka/nodejs-kafka/README.adoc
+:getting-started-rhoas-cli-url-kafka: kafka/rhoas-cli-getting-started-kafka/README.adoc
+:topic-config-url-kafka: kafka/topic-configuration-kafka/README.adoc
+:consumer-config-url-kafka: kafka/consumer-configuration-kafka/README.adoc
+:access-mgmt-url-kafka: kafka/access-mgmt-kafka/README.adoc
+:metrics-monitoring-url-kafka: kafka/metrics-monitoring-kafka/README.adoc
+:service-binding-url-kafka: kafka/service-binding-kafka/README.adoc
+:message-browsing-url-kafka: kafka/message-browsing-kafka/README.adoc
+
+//OpenShift Service Registry
+:product-long-registry: OpenShift Service Registry
+:product-registry: Service Registry
+:registry: Service Registry
+:product-version-registry: 1
+:service-url-registry: https://console.redhat.com/application-services/service-registry/
+:getting-started-url-registry: registry/getting-started-registry/README.adoc
+:quarkus-url-registry: registry/quarkus-registry/README.adoc
+:getting-started-rhoas-cli-url-registry: registry/rhoas-cli-getting-started-registry/README.adoc
+:access-mgmt-url-registry: registry/access-mgmt-registry/README.adoc
+:content-rules-registry: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/9b0fdf14-f0d6-4d7f-8637-3ac9e2069817[Supported Service Registry content and rules]
+:service-binding-url-registry: registry/service-binding-registry/README.adoc
+
+//OpenShift Connectors
+:connectors: Connectors
+:product-long-connectors: OpenShift Connectors
+:product-connectors: Connectors
+:product-version-connectors: 1
+:service-url-connectors: https://console.redhat.com/application-services/connectors
+:getting-started-url-connectors: connectors/getting-started-connectors/README.adoc
+:getting-started-rhoas-cli-url-connectors: connectors/rhoas-cli-getting-started-connectors/README.adoc
+
+//OpenShift API Designer
+:product-long-api-designer: OpenShift API Designer
+:product-api-designer: API Designer
+:product-version-api-designer: 1
+:service-url-api-designer: https://console.redhat.com/application-services/api-designer/
+:getting-started-url-api-designer: api-designer/getting-started-api-designer/README.adoc
+
+//OpenShift API Management
+:product-long-api-management: OpenShift API Management
+:product-api-management: API Management
+:product-version-api-management: 1
+:service-url-api-management: https://console.redhat.com/application-services/api-management/
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-using-iac-tools"]
+= Using Infrastructure as Code tools with Kafka instances in {product-long-kafka}
+ifdef::context[:parent-context: {context}]
+:context: using-iac-tools
+
+[role="_abstract"]
+As a developer of applications and services, you can use Infrastructure as Code (IaC) tools to automate deployment and management of architectures.
```

Comment:
I wonder if this functionality is targeted more to the administrator persona, than developers. To my mind, it would be administrators that want to automate the deployment and management of Kafka instances? That might be something that's described in the Epic Brief for this feature.